docs/gravitino-server-config.md

```diff
@@ -138,6 +138,23 @@ The plugin provides several operational modes for how to process event, supporti
 
 For more details, please refer to the definition of the plugin.
 
+### Audit log configuration
+
+The audit log framework defines how metadata changes are logged to various storages. The formatter defines an interface that transforms different `Event` types into a unified `AuditLog`.
```

Comment:
The audit log framework defines how metadata changes are logged to various storages -> The audit log framework defines how audit logs are formatted and written to various storages?